src/Gtk/Perspex.Cairo/Media/RadialGradientBrushImpl.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using global::Cairo;
+
+namespace Perspex.Cairo
+{
+	public class RadialGradientBrushImpl : BrushImpl
+	{
+		public RadialGradientBrushImpl(Perspex.Media.RadialGradientBrush brush, Size destinationSize)
+		{
+			var center = brush.Center.ToPixels(destinationSize);
```

Comment:
I see you do a null check on brush in Perspex.Direct2D/Media/RadialGradientBrushImpl.cs but not one here, why?